README.rst

```diff
@@ -6,23 +6,18 @@ Pure Python tools for reading and writing all TIFF IFDs, sub-IFDs, and tags.
 
 Developed by Kitware, Inc. with funding from The National Cancer Institute.
 
-Example
-=======
+Installation
+============
 
-.. code-block:: python
+``tifftools`` is available on PyPI and conda-forge.
 
-    import tifftools
-    info = tifftools.read_tiff('photograph.tif')
-    info['ifds'][0]['tags'][tifftools.Tag.ImageDescription.value] = {
-        'data': 'A dog digging.',
-        'datatype': tifftools.Datatype.ASCII
-    }
-    exififd = info['ifds'][0]['tags'][tifftools.Tag.EXIFIFD.value]['ifds'][0][0]
-    exififd['tags'][tifftools.constants.EXIFTag.FNumber.value] = {
-        'data': [54, 10],
-        'datatype': tifftools.Datatype.RATIONAL
-    }
-    tifftools.write_tiff(info, 'photograph_tagged.tif')
+To install with pip from PyPI::
+
+    pip install tifftools
+
+To install with conda::
+
+    conda install -c conda-forge tifftools
```

Comment:
We should add that IF you want to specify projections other than EPSG, that you can install with the [geo] extra.  We can add that without that this library has no external dependencies.

Reply:
Added in [1c9996c](https://github.com/DigitalSlideArchive/tifftools/pull/104/commits/1c9996c46afbf5a839657e1799ea4057b4b784a5)